packages/synapse-interface/utils/time.ts

```diff
@@ -16,3 +26,29 @@ export const formatDate = (date) => {
 export const formatTimestampToDate = (timestamp) => {
   return formatDate(new Date(timestamp * 1000))
 }
+
+export const convertUnixTimestampToMonthAndDate = (
```

Comment:
heads up this is built into js w/ `toLocaleString`:

```
export const convertUnixTimestampToMonthAndDate = (unixTimestamp: number): string => {
  const date = new Date(unixTimestamp * 1000)
  return date.toLocaleString('en-US', { month: 'short', day: 'numeric' })
}
```

Reply:
thank you! this is alot better - updated in https://github.com/synapsecns/sanguine/pull/1243/commits/33afb126536e90448b8604f99e53cb26b30a6104